### PR TITLE
updated contacts

### PR DESCRIPTION
**Pull request by prashant280920**
updated contact pages #32 @vsvipul @Signior-X

### CODE REVIEW

**Comment by vsvipul** [comment]
@prashant280920  attach screenshots with  every PR u make.

File: contacts.html [hunk 1]
```diff
@@ -304,219 +304,219 @@ <h2>Contacts</h2>
...
 						<tr>
 							<td class='tg-i6ua'>Programming Club<br></td>
+							<td class='tg-03to'>pc@iitmandi.ac.in</td>
 							<td class='tg-03to'>Subhash Suman<br></td>
 							<td class='tg-03to'>b18088@students.iitmandi.ac.in</td>
 							<td class='tg-03to'>7703813759</td>
-							<td class='tg-03to'></td>
+							
 							<td class='tg-kr4b'>Dr. Sreelakshmi Manjunath</td>
-							<td class='tg-03to'>ssreelakshmi@iitmandi.ac.in</td>
```
Comment (Signior-X) [inline contacts.html:323]: @prashant280920 Have you removed the Email ids??

**Review by Signior-X — COMMENTED** [review]
Provide a screenshot of work and you haven't removed the Email Adresses na?

File: contacts.html (hunk 1 above)
Comment (Signior-X) [reply contacts.html:323]: Did you Added the club emails?

**Comment by prashant280920** [comment]
wait i will add the screenshot

**Comment by prashant280920** [comment]
Technical contacts

cultural contacts

research contacts

**Comment by Signior-X** [comment]
I feel this is quite good!
The emails are long... but I guess we can't do anything for that as of now..
Rest changes are done as suggested.
Nice @prashant280920

**Comment by Signior-X** [comment]
Fixes #32

**Review by Signior-X — COMMENTED** [review]
All changes are good!

**Pull request merged**